src/Tabs/TabTemplate.js

```diff
@@ -1,29 +1,29 @@
-import React, {Component, PropTypes} from 'react';
+import React, {PropTypes} from 'react';
 
-class TabTemplate extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    selected: PropTypes.bool,
-  };
+const styles = {
+  width: '100%',
+  position: 'relative',
+  textAlign: 'initial',
+};
 
-  render() {
-    const styles = {
-      width: '100%',
-      position: 'relative',
-      textAlign: 'initial',
-    };
+const TabTemplate = ({children, selected, style}) => {
+  const templateStyle = Object.assign({}, styles, style);
+  if (!selected) {
+    templateStyle.height = 0;
+    templateStyle.overflow = 'hidden';
```

Comment:
I'm wondering why we don't use `display: 'none'`. But I wouldn't change it now. I'm  too afraid about the side effect 😁 .